app/views/users/index.html.erb

```diff
@@ -4,7 +4,18 @@
       <li>
         Name: <%= user.name %>
         <span class="profile-link">
+
           <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
+          
+          <% if current_user != User.find(user.id)%>
+            <% if show_add_friend(user) %>
+              <%= link_to 'Invite to friendship', {:controller => 'friendships', :action => 'create', :id => user.id}, method: :post, class: 'profile-link' %>
+            <% end %>
+
+            <% if show_already_friend(user) %>
+              <span> | you are friends</span>
+            <% end %>
+          <% end %>
```

Comment:
Extract the logic in Users view into helper methods.

Reply:
Issue solved